### PR TITLE
Posting jobsto db

### PR DESCRIPTION
**Pull request by amrathmd**

### CODE REVIEW

**Review by maharjunm — DISMISSED** [review]

File: src/pages/Home/Home.tsx [hunk 1]
```diff
@@ -11,18 +11,41 @@ import data from '../../components/SearchBar/data';
 const Home = () => {
   const [currentJob,setCurrentJob]= useState(null);
   const [view,setView]= useState('hide');
+  const [jobs,setJobs]=React.useState([]);
+  const [selectedJob,setSelectedJob]=React.useState('');
+  const [job,setJob] =React.useState([]);
+  React.useEffect(()=>{
+    const fetchData=async()=>{
+      await axios.get('http://localhost:3000/v1/job')
```
Comment (maharjunm) [inline src/pages/Home/Home.tsx]: read it from .env.

File: src/pages/PostJobs/Form.tsx [hunk 2]
```diff
@@ -74,14 +134,14 @@ const Form = () => {
       }, 2000);
     }
   };
-
-  const onSubmit = (e:any) =>{
-    e.preventDefault();
+  const onSubmit=async()=>{
+    const res=await axios.post('http://localhost:3000/v1/job',form).then(res=>console.log(res)).catch(e=>console.log(e));
```
Comment (maharjunm) [inline src/pages/PostJobs/Form.tsx]: read ip from .env file.

File: src/pages/Home/Home.tsx (hunk 1 above)
Comment (amrathmd) [reply src/pages/Home/Home.tsx]: solved

File: src/pages/PostJobs/Form.tsx (hunk 2 above)
Comment (amrathmd) [reply src/pages/PostJobs/Form.tsx]: solved

**Review by maharjunm — APPROVED** [review]

**Pull request merged**